### PR TITLE
Update standard

### PR DESCRIPTION
**Pull request by weshaggard**
PTAL @ericstj @joperezr

### CODE REVIEW

File: src/System.ValueTuple/ref/Configurations.props [hunk 1]
```diff
@@ -4,8 +4,6 @@
     <BuildConfigurations>
       netcoreapp;
       uap;
-      netstandard;
-      netstandard1.0;
```
Comment (ericstj) [inline src/System.ValueTuple/ref/Configurations.props]: What ref is used during API compat for the netstandard1.0 build of ValueTuple?

File: src/System.ValueTuple/ref/System.ValueTuple.csproj [hunk 2]
```diff
@@ -34,7 +34,7 @@
   <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.0'">
     <Reference Include="System.Runtime" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'portable_net40+sl4+win8+wp8'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'portable_net40+sl4+win8+wp8' or '$(TargetGroup)' == 'netfx'">
```
Comment (ericstj) [inline src/System.ValueTuple/ref/System.ValueTuple.csproj]: I don't see a netfx configuration of this project.

File: src/System.ValueTuple/src/Configurations.props [hunk 3]
```diff
@@ -4,7 +4,6 @@
     <BuildConfigurations>
       portable_net40+sl4+win8+wp8;
       netstandard1.0;
```
Comment (weshaggard) [inline src/System.ValueTuple/src/Configurations.props]: Yes I just added netfx here.

File: src/System.ValueTuple/ref/Configurations.props (hunk 1 above)
Comment (weshaggard) [inline src/System.ValueTuple/ref/Configurations.props]: Currently it was already disabled in this project for anything but netcoreapp and uap. I will consider re-enabling it and using the

Comment (ericstj) [inline src/System.ValueTuple/ref/Configurations.props]: I see.

File: src/System.ValueTuple/src/Configurations.props (hunk 3 above)
Comment (ericstj) [inline src/System.ValueTuple/src/Configurations.props]: I see.

**Comment by ericstj** [comment]
/cc @jcouv 
Now that ValueTuple is part of netstandard.library it cannot have a nestandard2.0 based implementation.  Instead we need to cross-compile for desktop.

@weshaggard I think we may want to make `netfx` mean `net461`.

**Comment by stephentoub** [comment]
Thanks, @weshaggard.

**Pull request merged**